recipe/build-mpi.sh

```diff
@@ -83,12 +82,11 @@ fi
             --with-hwloc=$PREFIX \
             --with-libevent=$PREFIX \
             --with-zlib=$PREFIX \
-            --enable-mca-dso \
```

Comment:
@minrk I seems it works w.r.t. CUDA. Am I missing something?

Reply:
It looks like cuda components are DSOs by default in openmpi 5: https://github.com/open-mpi/ompi/pull/12055 (also [reflected in build output](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1063030&view=logs&j=d818e662-7181-5ce9-b265-50d03742b7af&t=dc9b07c2-72db-585f-80fc-2b606081ad2b&l=15237), which shows all the components that link libcuda are in DSOs).

but the docs haven't been updated to reflect that: https://github.com/open-mpi/ompi/issues/12911